soda/contracts/docs/README.md

```diff
@@ -0,0 +1,81 @@
+Contracts will significantly reduce the number of data issues that occur and it will
+help the diagnostic process.
+
+## 1) Writing contract YAML files
+
+> Prerequisites: To write contract YAML files, you will need:
+> * A YAML text editor
+> * Optional: a local development environment to test contract YAML files
+> * Optional: a git repository to version control your contract YAML file
+
+See chapter [writing contract  YAML files](01_writing_contract_yaml_files/README.md) on how to write schema and
+other data quality checks in Soda contract YAML files
+
+## 2) Choose a way to verify a contract
+
+> Prerequisites.  To verify a contract, you need:
+> * A Soda contract YAML file
+> * All connection details to your SQL engine like Snowflake or postgres
+
+Choose the most way for you to verify your contract. The solutions to verifying a contract
+below can be used to build your workflow for:
+* Verifying a contract in production (when new data arrives)
+* Verifying a contract in CI/CD (when pipeline code changes)
+* Verifying a contract on a fixed time schedule (not recommended)
+
+To make a choice, consider what triggers execution of your contract verification and the in what
+environment will it run?  Eg A shell script in your local development environment using the Soda
+contract API in a Python virtual environment, an Airflow operator in a DAG that is executed, a
+GitHub commit, etc?
+
+* [Verifying a contract in Python](02_verifying_a_contract_in_python/README.md):
+  This is the most basic, common and versatile way to verify a contract.  In fact, all other ways below
+  are based on this Python library approach.  To use our Soda Contracts Python library and API you
+  have to know Python like setting up a Python virtual environment and Python library dependencies.
+  Instructions are provided.
+
+* [Verifying a contract using docker (Roadmap)](03_verifying_a_contract_using_docker/README.md):
+  If you know how to work with docker containers, then the Soda contract docker container removes the
+  hassle of setting up the virtual environment and ensuring that the versions of all the libraries are
+  compatible.
+
+* [Verifying contracts in an Airflow DAG (Roadmap)](04_verifying_a_contract_in_an_airflow_dag/README.md):
+  If you want to run a contract verification in an Airflow DAG, these instructions show how you can add
+  an operator to your Airflow DAG.
+
+* [Verifying a contract using GitHub action (Roadmap)](05_verifying_a_contract_as_a_github_action/README.md):
+  If you want to run a contract verification as part of you CI/CD workflow in GitHub actions after each
+  commit on a PR, then check out this Soda github actions support.
+
+* [Verifying contracts using Soda Agent (Roadmap)](06_verifying_contracts_using_soda_cloud_agent/README.md):
+  Running contract verification on a the Soda Agent managed service is the easiest.  You can use an API
+  and don't have to manage contract verification.
+
+## 3) Enable contract check contributions
```

Comment:
This section goes away from generic OSS approach to Cloud a bit too quick imo, I would propose we first sell the idea of contribution, then stop at people committing different files into repositories etc, then go to Cloud as a remedy to that pain

Reply:
Well no.  This is part of the task based approach Janet started.  It's the last step of a high level set of tasks when implementing data contracts.  The last step is to enable contributions from Soda Cloud.  

But we should only document this when it's ready.  So it will be removed here until it's done.